src/App.tsx

```diff
@@ -2,18 +2,153 @@ import React from 'react';
 import './App.css';
 
 import users from './api/users';
+import todosFromServer from './api/todos';
 
-const App: React.FC = () => {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
-};
+interface Todo {
+  userId?: number | string;
+  id: number | string;
+  title: string;
+  user: string | null;
```

Comment:
create type for user
```suggestion
  user?: User | null;
```